sdk-extensions/incubator/src/main/java/io/opentelemetry/sdk/extension/incubator/fileconfig/ResourceFactory.java

```diff
@@ -115,4 +140,52 @@ private int order() {
       return order;
     }
   }
+
+  private static boolean matchAll(String attributeKey) {
+    return true;
+  }
+
+  private static Predicate<String> detectorAttributeFilter(
+      @Nullable DetectorsModel detectorsModel) {
+    if (detectorsModel == null) {
+      return ResourceFactory::matchAll;
+    }
+    DetectorAttributesModel detectorAttributesModel = detectorsModel.getAttributes();
+    if (detectorAttributesModel == null) {
+      return ResourceFactory::matchAll;
+    }
+    List<String> included = detectorAttributesModel.getIncluded();
+    List<String> excluded = detectorAttributesModel.getExcluded();
+    if (included == null && excluded == null) {
+      return ResourceFactory::matchAll;
+    } else if (included == null) {
+      return excludedPredicate(excluded);
+    } else if (excluded == null) {
+      return includedPredicate(included);
+    }
```

Comment:
Prefer flatter with the redundant `else`s removed.
```suggestion
    }
    if (included == null) {
      return excludedPredicate(excluded);
    }
    if (excluded == null) {
      return includedPredicate(included);
    }
```

Reply:
Me too!